accumulator/accumulate_signals_activities.go

```diff
@@ -0,0 +1,25 @@
+package accumulator
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"go.temporal.io/sdk/activity"
+)
+
+// this activity will process all of the signals together
+func ComposeGreeting(ctx context.Context, s []AccumulateGreeting) (string, error) {
+	log := activity.GetLogger(ctx)
+	if(len(s) < 1) {
```

Comment:
Please make sure `go fmt` is run on all Go files (most editors do it on save by default). This is a bit off as is the lack of blank line separating stdlib imports from non-stdlib. The formatting is off on a lot of code in this PR.

Reply:
done thanks